pkg/resource/caconfig.go

```diff
@@ -60,17 +66,34 @@ func (gcac *generatorCAConfig) GetName() string {
 func (gcac *generatorCAConfig) expected() (runtime.Object, error) {
 	cm := &corev1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:        gcac.GetName(),
-			Namespace:   gcac.GetNamespace(),
-			Annotations: map[string]string{"service.alpha.openshift.io/inject-cabundle": "true"},
+			Name:      gcac.GetName(),
+			Namespace: gcac.GetNamespace(),
 		},
 		Data:       map[string]string{},
 		BinaryData: map[string][]byte{},
 	}
 
+	serviceCA, err := gcac.lister.Get(gcac.serviceCAName)
+	if errors.IsNotFound(err) {
+		glog.Infof("missing the service CA configmap: %s", err)
+	} else if err != nil {
+		return cm, err
+	} else {
+		if cert, ok := serviceCA.Data["service-ca.crt"]; ok {
+			internalHostname, err := getServiceHostname(gcac.serviceLister, gcac.serviceName)
+			if err != nil {
+				return cm, err
+			}
+
+			cm.Data[internalHostname] = cert
+		} else {
```

Comment:
+1 - @dmage this will also help us drop the port on the internal registry's service.